core/templates/domain/topic/editable-topic-backend-api.service.spec.ts

```diff
@@ -16,110 +16,101 @@
  * @fileoverview Unit tests for EditableTopicBackendApiService.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
-
-import { TranslatorProviderForTests } from 'tests/test.extras';
-
-require('domain/topic/editable-topic-backend-api.service.ts');
-require('services/csrf-token.service.ts');
-
-describe('Editable topic backend API service', function() {
-  var EditableTopicBackendApiService = null;
-  var sampleDataResults = null;
-  var $httpBackend = null;
-  var CsrfService = null;
-
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(
-    angular.mock.module('oppia', TranslatorProviderForTests));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+import { EditableTopicBackendApiService } from
```

Comment:
Add a newline above, separate the Angular imports and the Oppia imports. Also, arrange these alphabetically.(CsrfTokenService import comes first)

Reply:
done

Reply:
Done